edge-hub/docker/linux/arm32v7/Dockerfile

```diff
@@ -1,5 +1,21 @@
-ARG base_tag=1.0.6.15-linux-arm32v7
-FROM azureiotedge/azureiotedge-hub-base:${base_tag}
+ARG base_tag=3.1.19-bionic-arm32v7
+FROM mcr.microsoft.com/dotnet/aspnet:${base_tag}
+
+# Add an unprivileged user account for running Edge Hub
+ARG EDGEHUBUSER_ID=13623
```

Comment:
Is 13623 a randomly picked number or is it magic?

Please consider elaborating in comment.

Reply:
https://github.com/Azure/iotedge/pull/4998

The original problem was that the default user was 1000 - that typically corresponds to the first user created on a Linux system, and may not be a user which should have R/W permission on edgeAgent/edgeHub files. The proposed fix was to pick a semi-arbitrary number above 10000 to avoid typical regular users, and 13622 and 13623 were chosen for edgeAgent and edgeHub, respectively.  I can elaborate this comment.

Reply:
@darobs 
I thought the hubStart.sh script executed as part of the container already does that?